pkg/config/msp_test.go

```diff
@@ -94,6 +108,799 @@ func TestMSPToProtoFailure(t *testing.T) {
+											"organizational_unit_identifiers": [],
+											"revocation_list": [],
+											"root_certs": [],
+											"signing_identity": {
+												"private_signer": {
+													"key_identifier": "",
+													"key_material": null
+												},
+												"public_signer": "LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCi0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K"
+											},
+											"tls_intermediate_certs": [],
+											"tls_root_certs": []
+										},
+										"type": 0
+									},
+									"version": "0"
+								}
+							},
+							"version": "0"
+						}
+					},
+					"mod_policy": "Admins",
+					"policies": {
+						"Admins": {
+							"mod_policy": "Admins",
+							"policy": {
+								"type": 3,
+								"value": {
+									"rule": "MAJORITY",
+									"sub_policy": "Admins"
+								}
+							},
+							"version": "0"
+						},
+						"Readers": {
+							"mod_policy": "Admins",
+							"policy": {
+								"type": 3,
+								"value": {
+									"rule": "ANY",
+									"sub_policy": "Readers"
+								}
+							},
+							"version": "0"
+						},
+						"Writers": {
+							"mod_policy": "Admins",
+							"policy": {
+								"type": 3,
+								"value": {
+									"rule": "ANY",
+									"sub_policy": "Writers"
+								}
+							},
+							"version": "0"
+						}
+					},
+					"values": {
+						"ACLs": {
+							"mod_policy": "Admins",
+							"value": {
+								"acls": {
+									"acl1": {
+										"policy_ref": "hi"
+									}
+								}
+							},
+							"version": "0"
+						},
+						"Capabilities": {
+							"mod_policy": "Admins",
+							"value": {
+								"capabilities": {
+									"V1_3": {}
+								}
+							},
+							"version": "0"
+						}
+					},
+					"version": "0"
+				}
+			},
+			"mod_policy": "",
+			"policies": {},
+			"values": {},
+			"version": "0"
+		},
+		"sequence": "0"
+	}
+	`
+
+	expectedConfigProto := &cb.Config{}
+	err = protolator.DeepUnmarshalJSON(bytes.NewBufferString(expectedConfig), expectedConfigProto)
+	gt.Expect(err).NotTo(HaveOccurred())
+
+	gt.Expect(config).To(Equal(expectedConfigProto))
+}
+
+func TestRevokeCertificateFromMSPFailure(t *testing.T) {
+	t.Parallel()
+	gt := NewGomegaWithT(t)
+
+	channelGroup, err := baseApplicationChannelGroup()
+	gt.Expect(err).ToNot(HaveOccurred())
+	config := &cb.Config{
+		ChannelGroup: channelGroup,
+	}
+
+	// update Org1 with the CRL
+	crlBytes := createCRLBytesForTest()
+	org, err := getApplicationOrg(config, "Org1")
+	gt.Expect(err).NotTo(HaveOccurred())
+	fabricMSPConfig, err := getFabricMSPConfig(org)
+	gt.Expect(err).NotTo(HaveOccurred())
+	fabricMSPConfig.RevocationList = append(fabricMSPConfig.RevocationList, crlBytes)
+	err = addMSPConfigToOrg(org, fabricMSPConfig)
+	gt.Expect(err).NotTo(HaveOccurred())
+	config.ChannelGroup.Groups[ApplicationGroupKey].Groups["Org1"] = org
+
+	cert := &x509.Certificate{
+		KeyUsage: x509.KeyUsageKeyAgreement,
+	}
+
+	fakeFailingSigner := &mock.Signer{}
```

Comment:
So this seems to be where a fake signer is used.  You can still cause a failure by changing the curve param ‘N’ value to the zero value.

Reply:
Removed this test for now since we're using the standard library call and not directly calling signer.Sign any more.